index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Matee Safi - Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <section class="header">
+      <header class="nav">
+        <nav>
+            <div class="logo-container">
+                <label class="logo-text">Matee Safi</label>
+            </div>
+            <div class="menu">
+                <img id="hamburger" src="./assets/hamburger-icon.svg" alt="hamburger menu">
+            </div>
+        </nav>
+      </header>
+    </section>
```

Comment:
- I believe it will be better if we left only `<header>` tag instead of having it inside a section. Can we kindly remove the `<section class="header">` . The `<header class="nav"> tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. 🤗